src/main/java/mate/academy/ConnectionUtil.java

```diff
@@ -0,0 +1,39 @@
+package mate.academy;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.SQLException;
+import java.util.Properties;
+
+public class ConnectionUtil {
+    private static String password;
+    private static final String DB_URL = "jdbc:mysql://localhost:3306/jdbc_intro";

```

Comment:
The DB_URL seems to be incomplete. Make sure to provide the full database URL including the database name, parameters, and avoid using schema names directly in the URL as per the checklist.